apps/yapms/src/routes/+page.svelte

```diff
@@ -1,20 +1,42 @@
-<div class="navbar bg-base-200">
-	<h1 class="text-lg font-bold m-auto">Yet Another Political Map Simulator</h1>
+<script>
+	import HomePageSidebar from "$lib/components/homePage/homePageSidebar.svelte";
+	import MapSelectionGrid from "$lib/components/homePage/mapSelectionGrid.svelte";
+	import MapSelectionTitle from "$lib/components/homePage/mapSelectionTitle.svelte";
+	import Login from '$lib/icons/Login.svelte';
+	import mapSelectSections from '$lib/assets/other/MapSelection.json';
+	import MapSelectionCard from "$lib/components/homePage/MapSelectionCard.svelte";
+	import MoreMapsModal from "$lib/components/modals/moremapsmodal/MoreMapsModal.svelte";
+
+</script>
+<div class="navbar bg-base-200 h-[10%]">
+	<div class="navbar-start"></div>
+	<div class="navbar-center">
+		<h1 class="text-2xl font-bold m-auto hidden lg:inline">Yet Another Political Map Simulator</h1>
+		<h1 class="text-2xl font-bold m-auto inline lg:hidden">YAPms</h1>
+	</div>
+	<div class="navbar-end">
+		<button class="btn px-8 btn-primary mr-2 hidden md:inline">Login</button>
+		<button class="btn btn-square mr-2 inline md:hidden"><Login class="h-8 m-auto"/></button>
+	</div>
 </div>
 
-<div class="grid grid-cols-2 gap-2 m-5">
-	<div class="card card-bordered w-96 bg-base-100 shadow-xl">
-		<div class="card-body">
-			<h2 class="card-title">United States</h2>
-			<div class="grid grid-cols-2 gap-2">
-				<a class="link" href="/app/usa/presidential/2022">President</a>
-				<a class="link" href="/app/usa/presidential/2022">Senate</a>
-				<a class="link" href="/app/usa/presidential/2022">House</a>
-				<a class="link" href="/app/usa/presidential/2022">Governor</a>
-			</div>
-			<div class="card-actions justify-end">
-				<button class="btn btn-sm btn-primary">Buy Now</button>
-			</div>
-		</div>
+<svelte:head>
+	<title>YAPms - Yet Another Political Map Simulator</title>
+</svelte:head>
```

Comment:
The svelte:head tag should go directly below the <script> tag. Just for organization.